skglm/solvers/__init__.py

```diff
@@ -1,2 +1,5 @@
-from .cd_solver import cd_solver_path  # noqa F401
-from .multitask_bcd_solver import multitask_bcd_solver_path  # noqa F401
+from .accelerated_cd import AcceleratedCD  # noqa F401
```

Comment:
rather than noqa'ing every line we can define `__all__ = [list of all classes imported in this file]`. I think it's the recommended solution